perfdash/google-gcs-downloader.go

```diff
@@ -47,26 +47,26 @@ type GoogleGCSDownloaderOptions struct {
 	OverrideBuildCount bool
 }
 
-// GoogleGCSDownloader that gets data about Google results from the GCS repository.
-type GoogleGCSDownloader struct {
-	GoogleGCSBucketUtils *bucketUtil
-	Options              *GoogleGCSDownloaderOptions
+// Downloader that gets data about Google results from the GCS repository.
+type Downloader struct {
+	BucketUtils BucketUtil
+	Options     *DownloaderOptions
 }
 
-// NewGoogleGCSDownloader creates a new GoogleGCSDownloader.
-func NewGoogleGCSDownloader(opt *GoogleGCSDownloaderOptions) (*GoogleGCSDownloader, error) {
+// NewDownloader creates a new Downloader.
+func NewDownloader(opt *DownloaderOptions) (*Downloader, error) {
```

Comment:
This is good, you could probably just make this accept the `BucketUtil` interface

```suggestion
func NewDownloader(opt BucketUtil) (*Downloader, error) {
```